tests/unit_tests/test_ecephys.py

```diff
@@ -32,3 +40,77 @@ def test_check_negative_spike_times_some_negative():
         object_name="Units",
         location="/",
     )
+
+
+class TestCheckElectricalSeries(TestCase):
+    def setUp(self):
+
+        nwbfile = NWBFile(
+            session_description="", identifier=str(uuid4()), session_start_time=datetime.now().astimezone()
+        )
+
+        device = nwbfile.create_device(name="dev")
+        group = nwbfile.create_electrode_group(
+            name="electrode_group", description="desc", location="loc", device=device
+        )
+
+        for _ in range(5):
+            nwbfile.add_electrode(
+                x=3.0,
+                y=3.0,
+                z=3.0,
+                imp=-1.0,
+                location="unknown",
+                filtering="unknown",
+                group=group,
+            )
+
+        self.nwbfile = nwbfile
```

Comment:
```suggestion
        cls.nwbfile = nwbfile
```
Only modification needed for `setUpClass` usage.